sources/packages/backend/apps/api/src/route-controllers/report/report.aest.controller.ts

```diff
@@ -47,6 +50,15 @@ export class ReportAESTController extends BaseController {
     @Body() payload: MinistryReportsFilterAPIInDTO,
     @Res() response: Response,
   ): Promise<void> {
+    // If the institution id received as a part of the payload is an empty string,
+    // then convert it to the appropriate number type before passing it to the SQL
+    // as parameter.
+    if (
+      payload.reportName === MinistryReportNames.StudentUnmetNeed &&
+      payload.params["institutionId"] === ""
+    ) {
+      payload.params["institutionId"] = -1;
+    }
```

Comment:
The `institutionId` can be used as an empty string in the SQL. Ideally, it shouldn't be manipulated like this.